src/features/ModelPlan/ModelToOperations/_components/ITSystemsTable/index.tsx

```diff
@@ -91,7 +97,8 @@ const ITSystemsTable = () => {
             milestones: [],
             facilitatedBy: [],
             neededBy: null,
-            status: null as any
+            status: null as any,
+            addedFromSolutionLibrary: false
```

Comment:
Can you explain why this uses a hardcoded value here instead of a result from the query please?

Reply:
Thanks Patrick! No that makes sense!